caas/kubernetes/provider/bootstrap.go

```diff
@@ -566,8 +566,11 @@ func (c *controllerStack) createControllerStatefulset() error {
 	numberOfPods := int32(1) // TODO(caas): HA mode!
 	spec := &apps.StatefulSet{
 		ObjectMeta: v1.ObjectMeta{
-			Name:        c.resourceNameStatefulSet,
-			Labels:      c.stackLabels,
+			Name: c.resourceNameStatefulSet,
```

Comment:
after doing this, the model operator in the controller namespace will be doing nothing.
Probably, we just do not deploy the model operator for the controller model?

Reply:
Hmmm seems like a lot of work for what could be used later on. There is nothing stopping people from deploying to controller model correct ?

Thoughts @wallyworld

Reply:
People can still deploy charms to the controller model. They do it all the time in IAAS; for k8s it's also possible but the need is much less. Still having the model operator there for when needed is a good thing. Its purpose it to track non-juju created resources; we know about the controller stateful set so no need to track it